crates/services/executor/src/executor.rs

```diff
@@ -441,6 +442,19 @@ where
         )?;
         debug_assert!(data.found_mint, "Mint transaction is not found");
 
+        executor_metrics()
```

Comment:
I put up a draft PR, because I'm not sure about how to approach this correctly. Please see the description of this PR: https://github.com/FuelLabs/fuel-core/pull/2323

Reply:
As per the convo, metrics have been moved to the importer and [this commit](https://github.com/FuelLabs/fuel-core/pull/2310/commits/37d4cdcc03897f1731560f4547393bc2eba57d99) makes the importer respect the `--metrics` parameter.